Apps/AdminWebClient/src/ClientApp/src/services/interfaces.ts

```diff
@@ -55,6 +57,22 @@ export interface ICommunicationService {
     delete(communication: Communication): Promise<void>;
 }
 
+export interface ICommunicationService {
+    initialize(http: IHttpDelegate): void;
+    add(communication: Communication): Promise<Communication>;
+    update(communication: Communication): Promise<void>;
+    getAll(): Promise<Communication[]>;
+    delete(communication: Communication): Promise<void>;
+}
```

Comment:
This interface appears to be a copy/paste remnant.  There is a duplicate definition above.